static/src/javascripts/projects/common/utils/$.js

```diff
@@ -0,0 +1,33 @@
+import bonzo from 'bonzo';
+import qwery from 'qwery';
+
+// Warning: side effect. This patches the bonzo module for use everywhere
+bonzo.aug({
+    height() {
+        return this.dim().height;
+    },
+});
+
+function $(selector, context) {
+    return bonzo(qwery(selector, context));
+}
+
+$.create = s => bonzo(bonzo.create(s));
+
+$.ancestor = (el, c) => {
+    if (el.nodeName.toLowerCase() === 'html') {
+        return false;
+    }
+    if (!el.parentNode || bonzo(el.parentNode).hasClass(c)) {
+        return el.parentNode;
+    }
+    return $.ancestor(el.parentNode, c);
+};
+
+$.forEachElement = (selector, fn) => {
+    const els = qwery(selector);
+    els.forEach(fn);
+    return els;
+};
```

Comment:
ooh yeah you're correct, i misread the docs, thought it returned an enhanced array-like object, but doesn't look like it all. will test it in the morning, but looks good!